CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+Merlin is open source software, and we welcome contribution!
+
+Our default branch is `master`. That's the stable branch, for users.
+
+`develop` is the branch you want to fork from and make pull requests into.
+
+Before issuing a pull request, please [squash your commits](https://github.com/LLNL/merlin/wiki/Squash-commits) down to one concise commit message.
```

Comment:
include a link to here: https://merlin.readthedocs.io/en/latest/merlin_developer.html

Reply:
Done.